tests/snapshots/TestCLICommands_atmos_terraform_help.stdout.golden

```diff
@@ -53,21 +53,29 @@ Flags:
         --                            Use double dashes to separate
                                       Atmos-specific options from native
                                       arguments and flags for the command.
+                                      
+                                      
 
         --append-user-agent string    Sets the TF_APPEND_USER_AGENT environment
                                       variable to customize the User-Agent
                                       string in Terraform provider requests.
                                       Example: 'Atmos/test (Cloud Posse;
                                       +https://atmos.tools)'. This flag works
                                       with almost all commands.
+                                      
+                                      
 
     -h, --help                        help for terraform (default "false")
+                                      
+                                      
 
         --skip-init                   Skip running 'terraform init' before
                                       executing the command (default "false")
+                                      
+                                      
 
     -s, --stack string                atmos terraform <terraform_command>
-                                      <component> -s <stack>
+                                       -s
```

Comment:
Need to check this 🤔

Reply:
interesting on cli and temp.log
This is printed correct. But not during generating this test